webapi/_test_data/all_data/permissions-data.ttl

```diff
@@ -273,6 +273,29 @@
                       knora-base:hasPermissions "CR knora-base:Creator|M knora-base:ProjectMember|V knora-base:KnownUser|RV knora-base:UnknownUser"^^xsd:string .
 
 
+### Default Object Access Permissions on beol:figure resource class
+<http://rdfh.ch/permissions/0801/006-d2>
+
+                      rdf:type knora-base:DefaultObjectAccessPermission ;
+
+                      knora-base:forProject <http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF> ;
+
+                      knora-base:forResourceClass beol:figure ;
+
+                      knora-base:hasPermissions "CR knora-base:Creator|M knora-base:ProjectMember|V knora-base:KnownUser|V knora-base:UnknownUser"^^xsd:string .
```

Comment:
Shouldn't this be written as like "`CR knora-base:Creator|M knora-base:ProjectMember|V knora-base:KnownUser,knora-base:UnknownUser"^^xsd:string ."`?

https://docs.knora.org/paradox/02-knora-ontologies/knora-base.html#permissions

Reply:
both have the same effect at the end. But you are right according to

> Each permission abbreviation is followed by a space, then a comma-separated list of groups that the permission is granted to.

comma separated list of groups is better to be used.